### PR TITLE
bug issue template

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: labels.yml [hunk 1]
```diff
@@ -25,6 +28,7 @@
 - name: "affects/cost"
   color: 9E3E9E
   description: Issue affects cost
+# To be merged into `kind/bug`
 - name: "+bug"
   color: CDCC00
   description: Software behavior contrary to spec
```
Comment (mightyiam) [inline labels.yml:34]: After this is merged I will migrate all the `+bug` issues to the `kind/bug` label and then make a PR to remove `+bug`.

**Review by mightyiam — COMMENTED** [review]
I noticed that a software bug has a clear enough definition that a template could be made for it that is more concise than the problem template. So, here it is.

**Comment by gomain** [comment]
I don't see the benefit of a separate template for bugs, opposed to bugs being issued as problems (kind/bug)? Maybe we can generalize the problem template to incorporate some _expected vs actual_ scheme. Existence of a specification (formal or not) isn't the essence of a bug. The way I see it, if a system is behaving in unexpected ways. It is a problem that needs fixing. Regardless of whether the fix is educating the user or making changes to the system. Identifying an issue as a bug from the get go is making presumptions that it's the system that needs changing in order to fix the problem. Use of labels allows altering this decision later.

**Comment by mightyiam** [comment]
@gomain I'm trying to make using templates as easy as possible.

I certainly don't want misuse of them, such as using the bug template when the issue is not clearly a bug.

Yet, when it is clearly a bug, this template is quicker to fill in than the problem template.

Templates benefit from specificity. If we find a template that is very specific to what you're looking to file then we're more happy to use it.

Generalization is difficult for some people.

To reduce the risk of the bug template being used for not-certainly-bugs I gave it a more specific description.

**Comment by gomain** [comment]
> Yet, when it is clearly a bug, this template is quicker to fill in than the problem template.

This is the presumption I'm talking about.

> Templates benefit from specificity. If we find a template that is very specific to what you're looking to file then we're more happy to use it.
> 
> Generalization is difficult for some people.

Shall we wait for some more input on this?

---

More specifically, this section of the problem template
```
## Situation
> What situation do they face exactly?
```
could be adjusted to
```
## Current Situation
> What situation do they face exactly?

## Expectation
> How should the situation be?
```

**Pull request closed without merging**